07-documentation.md

```diff
@@ -0,0 +1,80 @@
+# REST documentation
+
+With the separation of frontend and backend it has become of critical importance that the REST interface is clearly documented. It is possible that frontend and backend tasks are performed by different developers, which simply means that the frontender should know what requests to call and what output to expect.
```

Comment:
I think it is `which requests to call and what kind of output the developer should expect.`